website/docs/r/app_service.html.markdown

```diff
@@ -172,7 +172,27 @@ The following arguments are supported:
 
 ~> **NOTE:** MySQL In App is not intended for production environments and will not scale beyond a single instance. Instead you may wish [to use Azure Database for MySQL](/docs/providers/azurerm/r/mysql_database.html).
 
-* `linux_fx_version` - (Optional) Linux App Framework and version for the AppService, e.g. `DOCKER|(golang:latest)`.
+* `linux_fx_version` - (Optional) Linux App Framework and version for the AppService, Possible values include:
+    - "DOCKER|`<image:tag>` (e.g. DOCKER|golang:latest) for single Docker container deployment.
+    - "COMPOSE|`<base64encoded_docker_compose_file>` for multiple Docker containers deployment.
+
+~> **NOTE:** When you use third party tools to update your Docker Compose definitions, you can tell Terraform not to worry about those changes with this snippet :
```

Comment:
This might be better in the examples folder with a link to it from here.

Reply:
Yes I agree, what would you like the file to be called and how would you link it from here?

Reply:
by the examples folder we're referring to here: https://github.com/terraform-providers/terraform-provider-azurerm/tree/master/examples which we can then link to [from the documentation](https://github.com/terraform-providers/terraform-provider-azurerm/blob/master/website/docs/r/app_service.html.markdown) as we do for example [in the VM resource](https://github.com/terraform-providers/terraform-provider-azurerm/blame/master/website/docs/r/virtual_machine.html.markdown#L17)

Would you mind pulling this example out into the `examples/` folder to give a working example of how to use this here?